source/MaterialXRender/ImageHandler.h

```diff
@@ -279,7 +279,7 @@ class ImageHandler
     /// Clear the contents of the image cache.
     /// deleteImage() will be called for each cache description to
     /// allow derived classes to clean up any associated resources.
-    void clearImageCache();
+    virtual void clearImageCache();
```

Comment:
Merge error?  This was removed in a previous change, since the method is never overridden, and it's called from the destructor of a polymorphic class.

Reply:
It's required so that PyImageHandler can override it in a newly exposed method.

Reply:
I think that the change to PyImageHandler is an unintended merge artifact as well.  We removed that binding at the same time the “virtual” modifier was removed here.

Reply:
Reverted now.